tgui/packages/tgui/interfaces/Pacman.js

```diff
@@ -26,7 +26,7 @@ export const Pacman = (props, context) => {
   let fuelRatio = fuel_stored / fuel_cap;
   let tmpRatio = tmp_current / tmp_max; // temperature ratio
   let output_watts = output_set * power_gen;
-  let fuel_sec = Math.round(fuel_stored / fuel_usage);
+  let fuel_sec = Math.round(fuel_stored / fuel_usage) * 2; // Multiply by two, because one cycle is two seconds.
```

Comment:
Is it not better to multiply it inside the round() for a more accurate value?

Reply:
You're absolutely correct. I've gone ahead and moved it inside.